### DISCUSSION TITLE
Conversations [Multiple Speakers]

### DISCUSSION THREAD

**Discussion by hypersniper05** [JonathanFly/bark#93] [Ideas]
I did not see a option for multi conversations by two speakers. I believe the orginal bark has a example in its notebook folder. Just wondering if it would be supported in the near future?

**Comment by JonathanFly** [comment]
> I did not see a option for multi conversations by two speakers. I believe the orginal bark has a example in its notebook folder. Just wondering if it would be supported in the near future?

That's really just a quirk of the prompts, not anything that original bark software is doing differently than this.

Try writing something like this.

```
It's been a great ball game. Over to you, Jane. | Thanks, Chris. I couldn't be more excited for this rematch. These are my favorite players.
```

**Reply by hypersniper05** [reply]
I ended up just modifying the orginal notebook version. I like that I can specify the speakers voice for each person by name. I also created a little script that will allow me to reroll the output clip before combining them. 

https://github.com/suno-ai/bark/blob/main/notebooks/long_form_generation.ipynb

**Comment by JonathanFly** [comment]
Ohh, if you want, I could make this better. Maybe? I could retroactively stuff the old generations into the history, so maybe the back and forth sounds more natural (since they will be responding to the actual words, potentially). There isn't much space but you could get the end of a sentence, so the audio might naturally respond to like, "And that's why YOU ARE WRONG." -> New speaker "I'm wrong???" (and it sounds like they heard previous sentence.)
